indexes/core_guidelines/core_guidelines_index.json

```diff
@@ -0,0 +1,3284 @@
+[
```

Comment:
Did you generate this file? It'd be helpful to know where it came from or have the script that was used to generate it.

Reply:
As I said above, I made it manually. I grepped the CppCoreGuidelines.md for things that look like links to individual guidelines. I also added the section names for good measure, and then I reformatted them into the JSON. I wanted to make a script to generate it, but the document is surprisingly less script-friendly than I thought it would be. I will think about how to automate it and make the script in the future, thankfully the core guidelines don't seem to add new guidelines often.

Reply:
Ah, I see. Happy to work with this for now then. Briefly experimenting with the md file, it looks like just searching for level 2 and 3 headings seems to work well, minus some FAQs at the end, but maybe this misses some stuff.